src/client/particles.h

```diff
@@ -8,7 +8,7 @@
 #include "irr_ptr.h"
 #include "ISceneNode.h"
 #include "S3DVertex.h"
-#include "SMeshBuffer.h"
+#include "CMeshBuffer.h"
```

Comment:
Do we want to use `<...>` or `"..."` for Irrlicht includes?

When Luanti still was an external library, `<...>` was the obvious choice, but now `"..."` would seem like the better option.

(I think we should maybe write the rule for this down somewhere.)

Reply:
"..." seems right.